#Find custom located LIBX11, LIBXFT, and LIBXEXT includes
#December 08, 2015 by B. Hajgato (Vrije Universiteit Brussel)
--- configure.orig	2015-12-08 11:43:15.397124435 +0100
+++ configure	2015-12-08 11:47:42.530999154 +0100
@@ -2681,6 +2681,7 @@
 
     check_header "X11/Xlib.h" "" \
         ${finkdir:+$finkdir/include} \
+        $EBROOTLIBX11/include \
         /usr/include /usr/X11R6/include /usr/include/X11 /usr/openwin/include \
         /usr/local/include /usr/local/include/X11 /usr/local/X11R6/include \
         /usr/X11/include /usr/include/X11R5
@@ -2707,6 +2708,7 @@
 
     check_header "X11/Xft/Xft.h" "" \
         ${finkdir:+$finkdir/include} \
+        $xftlibdir/../include \
         /usr/include /usr/X11R6/include /usr/include/X11 /usr/openwin/include \
         /usr/local/include /usr/local/include/X11 /usr/local/X11R6/include \
         /usr/X11/include /usr/include/X11R5
@@ -2720,6 +2722,7 @@
     if test ! "x$arch" = "xaix5" ; then
         check_header "X11/extensions/shape.h" "" \
             ${finkdir:+$finkdir/include} \
+            $xextlibdir/../include \
             /usr/include /usr/X11R6/include /usr/include/X11 \
             /usr/openwin/include /usr/local/include /usr/local/include/X11 \
             /usr/local/X11R6/include /usr/X11/include /usr/include/X11R5
